files/gitlab-runner-stack.yml

```diff
@@ -34,8 +34,39 @@ parameters:
     description: >
       How big the volume attached to the runner VM should be.
     type: string
+  runner_ssh_key_rsa_private:
+    description: >
+      Preconfigured RSA host private key
+    type: string
+    default: ''
+  runner_ssh_key_rsa_public:
+    description: >
+      Preconfigured RSA host public key
+    type: string
+    default: ''
+  runner_ssh_key_ecdsa_private:
+    description: >
+      Preconfigured ECDSA host private key
+    type: string
+    default: ''
+  runner_ssh_key_ecdsa_public:
+    description: >
+      Preconfigured ECDSA host public key
+    type: string
+    default: ''
 
 resources:
+  cloud_config_ssh_keys:
+    # this resource demonstrates multiple cloud-config resources
```

Comment:
Does it? How? Not that familiar with cloud_config.

Reply:
Ah, 'demonstrates' is a giveaway (copy paste f+ modifications from an example). Fixing.